#goal definition
robot_module_msgs/JointCommand[] joints
---
#result definition
sensor_msgs/JointState final_joints
bool fully_succeeded
---
#feedback
sensor_msgs/JointState joints
